spec/linter-mypy_project_name-spec.coffee

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env coffee
+#
+# This spec file validates:
+#  * The use of $PROJECT_NAME variable in the config path
+#
+#  If it fails:
+#  * Adjust the project_name method
+
+LinterMyPystyle = require '../lib/init'
+path = require 'path'
+fs = require 'fs'
+{CompositeDisposable} = require 'atom'
+helpers = require 'atom-linter'
+
+fdescribe "The MyPy provider for Linter", ->
```

Comment:
fdescribe vs describe

There is an extra f at the beginning of the line... is it needed?